packages/dso-toolkit/src/components/icon/icon.stories-of.ts

```diff
@@ -40,9 +46,19 @@ export function iconMeta<TRenderer extends Renderer>({ readme }: MetaOptions = {
 export function iconStories<Implementation, Templates, TemplateFnReturnType>({
   storyTemplates,
   templateContainer,
+  decorator,
 }: IconStoriesParameters<Implementation, Templates, TemplateFnReturnType>): IconStories {
+  const icons = process.env.ICONS?.split(",");
   return {
-    Icon: {
+    Default: {
+      argTypes: iconArgTypes,
+      args: {
+        icon: "user",
+      },
+      render: templateContainer.render(storyTemplates, (args, { iconTemplate }) => iconTemplate(iconArgsMapper(args))),
+    },
+    Overview: {
+      decorators: [(story) => (decorator ? decorator(story, icons || []) : story)],
```

Comment:
Netter zou zijn om de lege array op regel 51 te maken. In de praktijk is er nu geen verschil, maar het is een ander verhaal wanneer `icons` vaker wordt gebruikt.